presto-phoenix/src/main/java/io/prestosql/plugin/phoenix/PhoenixClient.java

```diff
@@ -167,6 +167,15 @@ public void execute(ConnectorSession session, String statement)
         }
     }
 
+    @Override
+    protected boolean filterSchema(String schemaName)
+    {
+        if (schemaName == null) {
```

Comment:
null?

Reply:
I extracted the condition from the following. But I will remove and leave commit message if you agree on. 
https://github.com/prestosql/presto/blob/b52bcd71d4a3fc17fcb74eb3aa1bd291b4111128/presto-phoenix/src/main/java/io/prestosql/plugin/phoenix/PhoenixClient.java#L160